<launch>
  <arg name="robot_namespace" default="tank"/>
  <arg name="robot_type" default="tank"/>
  <node pkg="dutuuv_control" type="AccelController.py" name="accel_controller" output="screen" >
    <rosparam command="load" file="$(find dutuuv_control)/config/$(arg robot_type)/inertial.yaml" />
    <remap from="cmd_accel" to="$(arg robot_namespace)/cmd_accel"/>
    <remap from="cmd_force" to="$(arg robot_namespace)/cmd_force"/>
    <remap from="cmd_effort" to="$(arg robot_namespace)/thruster_manager/input"/>
  </node>
</launch>
